utils/utils.go

```diff
@@ -171,6 +175,23 @@ func GetIdentifiers(args *skel.CmdArgs) (workloadID string, orchestratorID strin
 	return workloadID, orchestratorID, nil
 }
 
+func GetHandleID(netName string, containerID string) (string, error) {
+	logger := log.WithFields(log.Fields{
+		"Network":     netName,
+		"ContainerID": containerID,
+	})
+	hasher := md5.New()
+	_, err := hasher.Write([]byte(netName))
+	if err != nil {
+		return "", err
+	}
+	hashedName := hex.EncodeToString(hasher.Sum(nil))
+	handleID := fmt.Sprintf("%s.%s", hashedName, containerID)
+	logger.Info("Generated handleID(%s) from hashed network name and containerID", handleID)
```

Comment:
Just a nit here - let's just use `WithFields` for all of them to make it consistent. 

Then, the log text should just be "Generated IPAM handle"

Also, it doesn't include the workload - we should pass in the main logger in order to get that context.

And finally, I think it should be a debug log.